packages/plugins/rubygem-radcli/rubygem-radcli.spec

```
@@ -64,24 +83,29 @@ rm -rf %{buildroot}%{gem_instdir}/ext/
 
 %files
 %dir %{gem_instdir}
-%{gem_extdir_mri}
-%{gem_instdir}/CHANGES
 %license %{gem_instdir}/LICENSE
 %{gem_instdir}/MANIFEST
 %{gem_instdir}/build_adcli.sh
-%exclude %{gem_instdir}/radcli.gemspec
 %{gem_instdir}/scripts
-%{gem_instdir}/lib
 %exclude %{gem_cache}
 %{gem_spec}
+%exclude %{gem_instdir}/radcli.gemspec
+%if 0%{!?scl:1}
+%{gem_extdir_mri}
+%{gem_instdir}/lib
```

This is equivalent to `%{gem_libdir}` in `/etc/rpm/macros.rubygems.rh-ruby25`

```suggestion
%{gem_libdir}
```